int32 vehicle_ID
bool started_ack